packages/ui/tailwind-utils-config/components/multi-select-v2.ts

```diff
@@ -1,3 +1,30 @@
+import { CSSRuleObject } from 'tailwindcss/types/config'
+
+const themes = ['danger'] as const
+
+function createMultiSelectThemeStyles() {
+  const styles: CSSRuleObject = {}
+
+  themes.forEach(theme => {
+    styles[`&:where(.cn-multi-select-${theme}):not(:has(input[disabled]))`] = {
+      borderColor: `var(--cn-border-${theme})`,
+      boxShadow: `var(--cn-ring-${theme})`,
+
+      '&:where(:hover)': {
+        borderColor: `var(--cn-border-${theme})`,
```

Comment:
if bordercolor doesn't change, do we need to provide it again on hover and on focus?